packages/lodestar/src/db/api/beacon/repositories/abstract.ts

```diff
@@ -211,16 +211,16 @@ export abstract class Repository<I extends Id, T> {
       lt: _encodeKey(this.bucket + 1, Buffer.alloc(0)),
     };
     if (opts) {
-      if (opts.lt) {
+      if (opts.lt !== undefined) {
```

Comment:
This wouldn't catch opts.lt = null.

It's better to use opts.lt && Number.isNumber(opts.lt) if you are trying to catch 0 in opts. But since we don't use negative numbers 0 here is same as undefined, right?

Reply:
> It's better to use opts.lt && Number.isNumber(opts.lt) if you are trying to catch 0 in opts
yeah I got an issue in `gt: 0` case, will fix like that

Reply:
Number.isInteger requires me to cast to number so I'd just compare to 0